Tests/XCUITests/BaseTestCase.swift

```diff
@@ -112,6 +112,136 @@ class BaseTestCase: XCTestCase {
         }
     }
 
+    func bookmark10Pages() {
+            navigator.openURL("google.com")
```

Comment:
A few comments about this method... 
- could we use the local website we have instead of real sites?
- I think it may be better to define a list and iterate over it instead of creating this long method. Something like:
```
let listWebsitesToBookmark = ["page1", "page2", ....]
bookmark10Pages(list) {
for site in list {
navigator.openURL(site)
waitUntilPageLoad()
}
}
```

Reply:
I guess we could use them but I didn't find a place where we have 10 different websites. Could you point them out?

Reply:
This is where we have the different html files with the local sites you could open, like this for example: https://github.com/mozilla-mobile/firefox-ios/blob/6d4b27db52f937e484af51ae1acd3c8f82ffcf79/Tests/XCUITests/FindInPageTest.swift#L86

Reply:
could do this with just 5 bookmarks? with a sample may be enough instead of having this super long test

Reply:
I guess we could do it with less but the scope of this steps is to see that the Recently Saved section has only 5 bookmarks and 5 reading list items displayed. I could add only 6 bookmarks and check if the correct number is displayed.